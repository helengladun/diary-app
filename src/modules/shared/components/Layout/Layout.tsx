import React, {Component} from 'react';
import {Sidebar} from "../Sidebar/Sidebar";

class Layout extends Component {
    render() {
        return (
            <div className="layout">
                <Sidebar/>
                <main className="content-container">
                    <div className="main-content">
                        {this.props.children}
                    </div>
                </main>
            </div>
        );
    }
}

export default Layout;