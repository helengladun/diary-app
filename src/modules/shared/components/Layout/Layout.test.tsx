import React from 'react';
import {shallow} from 'enzyme';
import Layout from './Layout';

describe('Layout component', () => {
    const wrapper = shallow(<Layout/>);

    it('make snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

});