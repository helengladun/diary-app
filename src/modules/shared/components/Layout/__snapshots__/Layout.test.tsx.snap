// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component make snapshot 1`] = `ShallowWrapper {}`;
